src/did_indy/driver/api/namespaces.py

```diff
@@ -39,6 +40,7 @@ async def get_info(ledgers: LedgersDep, store: StoreDep) -> NamespaceList:
                 namespace=namespace,
                 nym=nym,
                 did=f"did:indy:{namespace}:{nym}",
+                genesis_transaction=ledgers.ledgers[namespace].genesis_txns_cache or "",
```

Comment:
Should probably just make this loop iterate over keys and values and then use the value to access this genesis_txns_cache attribute